examples/axes_grid1/demo_axes_grid2.py

```diff
@@ -20,16 +20,15 @@ def get_demo_image():
     return z, (-3, 4, -4, 3)
 
 
-def add_inner_title(ax, title, loc, size=None, **kwargs):
+def add_inner_title(ax, title, loc, **kwargs):
```

Comment:
Note: I removed `size` here because it was unused and didn't want to leave it cluttering up the example.